com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md

```diff
@@ -70,11 +70,14 @@ From HDRP 12.0, The AOV API includes the following improvements:
 
 From HDRP 12.0, More Options have become Additional Properties. The way to access them has also changed. The cogwheel that was present in component headers has been replaced by an entry in the contextual menu. When you enable additional properties, Unity highlights the background of each additional property for a few seconds to show you where they are.
 
-### Path traced fabric material
+### More path traced materials
 
 ![](Images/HDRPFeatures-FabricPT.png)
 
-HDRP's path tracer now offers support for the fabric material, in both its cotton/wool and silk variants.
+HDRP's path tracer now offers support for additional materials:
+- Fabric, in both its cotton/wool and silk variants.
+- AxF, in both its SVBRDF and car paint variants.
+- Stacklit.
 
```

Comment:
HDRP path tracing now supports the following Materials:
- Fabric: Cotton/wool and silk variants.
- AxF: SVBRDF and car paint variants.
- Stacklit.